201-userdefined-routes-appliance/azuredeploy.json

```diff
@@ -0,0 +1,553 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters" : {
+      "location": {
+        "type": "string",
+		"allowedValues": ["East US", "West US", "North Central US", "South Central US", "Central US", "West Europe", "North Europe", "East Asia", "South East Asia", "Japan West", "Japan East"],
+        "metadata" : {
+          "Description" : "Deployment location"
+        }
+      },
+	  "newStorageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique DNS Name for the Storage Account where the Virtual Machine's disks will be placed."
+            }
+        },
+		"adminUsername": {
+            "type": "string",
+            "metadata": {
+               "description": "Username for the Virtual Machine."
+            }
+        },
+		"adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Password for the Virtual Machine."
+            }
+        },
+		"dnsNameForPubIP1": {
+            "type": "string",
+            "metadata": {
+                  "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+            }
+        },
+		"dnsNameForPubIP2": {
+            "type": "string",
+            "metadata": {
+                  "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+            }
+        },
+        "dnsNameForPubIP3": {
+            "type": "string",
+            "metadata": {
+                  "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+            }
+        },
+		"vNetAddressPrefix":{
+			"type" : "string",
+			"defaultValue" : "10.0.0.0/16",
+			"metadata" : {
+			  "Description" : "Address prefix"
+			}
+		},
+		"subnet1Prefix" : {
+			"type" : "string",
+			"defaultValue" : "10.0.0.0/24",
+			"metadata" : {
+			  "Description" : "Subnet 1 Prefix"
+			}
+		},
+		"subnet2Prefix" : {
+			"type" : "string",
+			"defaultValue" : "10.0.1.0/24",
+			"metadata" : {
+          "Description" : "Subnet 2 Prefix"
+			}
+		},
+		"subnet3Prefix" : {
+			"type" : "string",
+			"defaultValue" : "10.0.2.0/24",
```

Comment:
These prefixes could become variables to reduce the number of parameters a user needs to enter